tools/doc/type-parser.js

```diff
@@ -28,6 +28,8 @@ const customTypesMap = {
 
   'AsyncIterator': 'https://tc39.github.io/ecma262/#sec-asynciterator-interface',
 
+  'AsyncGeneratorFunction': 'https://tc39.es/ecma262/#sec-async-generator-function-definitions',
```

Comment:
It seems this file uses some logical sorting (general types, the Node.js types grouped by modules) with alphabetical sorting inside sections. Maybe this type can be joined with `AsyncIterator` type alphabetically:
```
  'AsyncGeneratorFunction': 'https://tc39.es/ecma262/#sec-async-generator-function-definitions',
  'AsyncIterator': 'https://tc39.github.io/ecma262/#sec-asynciterator-interface',
```

Reply:
Good point - agreed

Reply:
```suggestion
  'AsyncIterator': 'https://tc39.github.io/ecma262/#sec-asynciterator-interface',
```